openscap_daemon/cli_helpers.py

```diff
@@ -302,3 +302,27 @@ def summarize_standard_compliance_results(arf_source, result_list):
         result_json["Custom"] = {"XCCDF result": result}
 
         result_list.append(result_json)
+
+def get_list_of_profiles(sds_filepath):
```

Comment:
There already is a function for this. Please use that instead. It's called scrape_profiles or something like that.